packages/jats-xml/src/types/refType.ts

```diff
@@ -0,0 +1,53 @@
+/**
+ * https://jats.nlm.nih.gov/archiving/tag-library/1.3/attribute/ref-type.html
+ */
+export enum RefType {
```

Comment:
Need to do a better job at auto-creating these from the schema. That is worth a bit of time I think, but out of scope for this PR!